### ISSUE TITLE
Typography weights and opacity

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/360-ds#51]
The following rules use `--base-hsl` `#153634` with opacity that mean the contrast doesn't meet WCAG AAA . An opacity of 0.95 is the minimum needed to pass on a white background. 

```
src/components/02-elements/breadcrumbs/breadcrumbs.scss:18:    color: hsla(var(--base-hsl), .4);
src/components/02-elements/typography/typography.scss:16:  color: hsla(var(--base-hsl), .7);
src/components/02-elements/typography/typography.scss:96:  color: hsla(var(--base-hsl), .8);
src/components/03-components/upload-file/upload-file.scss:13:    color: hsla(var(--base-hsl), .2);
src/components/04-modules/filters/filters.scss:31:  &::-webkit-input-placeholder { color: hsla(var(--base-hsl), .7); }
src/components/04-modules/filters/filters.scss:32:  &:-moz-placeholder { color: hsla(var(--base-hsl), .7); }
src/components/04-modules/filters/filters.scss:33:  &::-moz-placeholder { color: hsla(var(--base-hsl), .7); }
src/components/04-modules/filters/filters.scss:34:  &:-ms-input-placeholder { color: hsla(var(--base-hsl), .7); }
src/components/04-modules/sidebar-blog/sidebar-blog.scss:43:  color: hsla(var(--base-hsl), .6);
src/components/04-modules/subscribe/subscribe.scss:31:  @include placeholder { color: hsla(var(--base-hsl), .9); }
src/components/04-modules/top-bar/top-bar.scss:198:    color: hsla(var(--base-hsl), .7);
```

Which is compounded / related to use of low font-weight in:

```
src/components/02-elements/typography/typography.scss:15:  font-weight: 300;
src/components/02-elements/typography/typography.scss:26:  font-weight: 300;
src/components/02-elements/typography/typography.scss:74:  font-weight: 300;
src/components/02-elements/typography/typography.scss:140:    font-weight: 300;
src/components/03-components/header-group/header-group.scss:35:  font-weight: 300;
src/components/04-modules/footer/footer.scss:118:    font-weight: 300;
src/components/04-modules/insights-card/insights-card.scss:32:    font-weight: 300;
src/components/04-modules/insights-card/insights-card.scss:68:    font-weight: 300;
src/components/04-modules/insights-card/insights-card.scss:107:  &__grant-n { font-weight: 300; }
src/components/04-modules/registry-card/registry-card.scss:32:  font-weight: 300;
src/components/04-modules/registry-card/registry-card.scss:52:  font-weight: 300;
src/components/04-modules/registry-card/registry-card.scss:95:.registry-grant__grant-n { font-weight: 300; }
**src/scss/pre/_reset.scss:21:  font-weight: 300;**
```

https://github.com/ThreeSixtyGiving/360-ds/blob/master/src/components/02-elements/typography/typography.scss
